go.mod

```diff
@@ -204,3 +205,5 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.3.0 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
+
+replace golang.org/x/exp => golang.org/x/exp v0.0.0-20230713183714-613f0c0eb8a1
```

Comment:
Just merged and forgot to add a comment about this, leaving it here for future reference. Can add it in the code if you prefer, too:

```
// Pin x/exp due to a breaking change (https://github.com/golang/exp/commit/302865e7556b4ae5de27248ce625d443ef4ad3ed#diff-ec22fd7f0aec27f1c6dc503e7dd3960c8f5ab781684218aa59b404ada0d48231R29)
// ```
// ../../go/pkg/mod/github.com/prometheus/prometheus@v0.47.0/model/labels/labels.go:365:23: type func(a Label, b Label) bool of func(a, b Label) bool {…} does not match inferred type func(a Label, b Label) int for func(a E, b E) int
// ```